### PR TITLE
motion 0.61.0 (new cask)

### PR DESCRIPTION
**Pull request by nihaals**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by nihaals** [comment]
I thought `:github_releases` already checked for the release marked latest but it seems it's picking up on the release created later instead. I tried searching the repo for examples but all the examples I could find were for skipping pre-releases so I'm not sure what the neatest way of finding the release is. See [the release](https://github.com/usemotion/desktopapp/releases/tag/v0.99.0) for more context.

File: Casks/m/motion.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  url "https://github.com/usemotion/desktopapp/releases/download/#{version}/motion-#{version}-mac-#{arch}.zip",
+      verified: "github.com/usemotion/desktopapp/"
+  name "Motion"
+  desc "To-do list and project management app which autoschedules tasks"
+  homepage "https://www.usemotion.com/"
+
+  livecheck do
+    url :url
+    strategy :github_releases
```
Comment (bevanjkay) [inline Casks/m/motion.rb:25]:
```suggestion
    strategy :github_latest
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/m/motion.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    strategy :github_latest
+  end
+
+  auto_updates true
+
+  app "Motion.app"
+
+  zap trash: [
+    "~/Library/Application Support/Motion",
+    "~/Library/HTTPStorages/com.electron.motion*",
```
Comment (reitermarkus) [inline Casks/m/motion.rb:34]:
```suggestion
    "~/Library/HTTPStorages/com.electron.motion",
```

**Review by reitermarkus — DISMISSED** [review]

File: Casks/m/motion.rb (hunk 2 above)
Comment (nihaals) [reply Casks/m/motion.rb:34]: Both `~/Library/HTTPStorages/com.electron.motion` and `~/Library/HTTPStorages/com.electron.motion.binarycookies` exist

**Comment by SMillerDev** [comment]
> I thought :github_releases already checked for [the release](https://github.com/usemotion/desktopapp/releases/tag/v0.99.0) marked latest but it seems it's picking up on the release created later instead. I tried searching the repo for examples but all the examples I could find were for skipping pre-releases so I'm not sure what the neatest way of finding the release is. See the release for more context.

That's `:github_latest`, releases just checks for the most recent release.

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @nihaals

**Review by miccal — APPROVED** [review]

**Pull request merged**